tests/lib/pcm_stream_channel_modifier/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
```

Comment:
2018 = 2023?

Reply:
This file originates from 2018, so I kept it.